internal/consumer/consumer.go

```diff
@@ -244,6 +244,11 @@ func (c *Consumer) shouldSkip(logger logger.Logger, evt *internal.DBChangeEvent)
 		found, valid, path, err := c.validator.Validate(*evt)
 		if err != nil {
 			logger.Error("error validating schema: %s", err)
```

Comment:
Does this result in both Error and Debug being logged?  Or does Go somehow handle this and only log the debug line if the condition matches?

Reply:
go catch just testing it...